DigitalLearningSolutions.Web/Views/Signposting/LearningHubLoginWarning.cshtml

```diff
@@ -30,6 +30,7 @@
 
     <form class="nhsuk-u-margin-bottom-3"
           method="post" novalidate
+          external-link-attributes="@(!Model.LearningHubAccountLinked)"

```

Comment:
So the effect of this is, "open in new tab if true" right?
`external-link-attributes` doesn't make that very clear to me. Why not call it `submit-in-new-tab`?

Reply:
Not strictly true, it set the attributes required but whether it opens in a new tab or not is down to users preference. But most likely new tab as most people I guess leave it as the default.
Yeah I get what you are saying by the name of `external-link-attributes` maybe `set-external-link-attributes` might of been a slightly better name.
I'm not convinced with `submit-in-new-tab`, partly because:
1. It can also be used on `<a>` tags and so its not always a `submit`
2. `submit...` is suggestive of some sort of internal submit behavior.  

Maybe `open-in-new-tab` ? 
But even then that is hiding the fact that the rel values of `noopener noreferrer` are being set. Which in the world of Javascript, for internal links, could cause an issue/confusion if `opener` was ever required.

`use-external-tab-opener` ?

Reply:
`use-external-tab-opener` works well I think. I think it's doubtful we'd ever want `opener`. If we're opening something in a new tab, the user has the history in the current tab, so they shouldn't need it in the new tab. Then there's also the security issue.

Reply:
Cool, that's done.